### ISSUE TITLE
No option to have collective on swapped channel

### ISSUE THREAD

**Issue by OpenUAS** [OpenUAS/DIY-Multiprotocol-TX-Module#1]
As user I want  to be able to use a DIY module in PPM use on a GRAUPNER/JR TX and be able to fly 3rd party protocol helis. For example on a MX22 when HELI model is chosen the Channel ordering changes from TAER to CAERxT  (where x is not in use per default). At the moment cannot be done with current state of code, but we want to be able. 

For now if it would work for the Heli protocols like WALKERA, DEVO that would be a good start

Note that one issue is that this transmitter can **NOT** remap channels of the PPM output. Trickery with mixing, will also not give a good solution.

One could avoid the Heli mode and mimic the behavious, unknown if it would work.

**Comment by OpenUAS** [comment]
{PLACEHOLDER} describe issue in detail

**Comment by OpenUAS** [comment]
WIP solution ramblings, not finished

#ifdef HAS_COLLECTIVE_CHANNEL
#define COLLECTIVE 0
#enndif

Do a big fat remap

  #undef AILERON
  #undef ELEVATOR
  #undef THROTTLE
  #undef RUDDER
  #define AILERON  1
  #define ELEVATOR 2
  #define THROTTLE 5
  #define RUDDER   3

That would not work since that is for the whole of the module, we only want it for a mode that is a HELI setting...

A variable thus must be changed dynamicly... Well lets use the define HAS_COLLECTIVE_CHANNEL in the DEVO code... Nicer would be in all of module code

Basically it is a new subprotocol, where we even have to change the input channel on the fly... 
Yuck, but OK

Let make some tryout solution where injection is most elegant and universally usable.
